docs/csv/example_files/ballot_selection.txt

```diff
@@ -1 +1,6 @@
-id,sequence_order
\ No newline at end of file
+id,sequence_order
```

Comment:
Ballot selection should be an abstract base class and not have any information associated with it itself. What particular use case did you see for having this be a separate file/class?